### PR TITLE
Colts, Revolvers, and Tasers

### PR DESCRIPTION
**Pull request by GhostActual**
Adds in a slew of new revolver sprites, colt sprites, and stun weapon sprites all from the old Armory Revamp PR. Replaces the Stun Revolver in the detective's locker with a Stun Revolver Box. Like the Detective Gun Box, it allows the player to pick a gun and name/change the sprite of it. Comments out the unused security version of the Xeno Taser and uses the blue taser sprite for the Xeno Taser. Fixes some errors that I caused with the ammo revamp by removing the ammo defines from `revolvers_vr` which caused sprite issues. Also cleans up the code in the files I touched a little.

### CODE REVIEW

File: code/game/objects/structures/crates_lockers/closets/walllocker.dm [hunk 1]
```diff
@@ -222,5 +222,37 @@
...
+	store_mobs = 0
+	wall_mounted = 1
+	plane = TURF_PLANE
+	layer = ABOVE_TURF_LAYER
+
+	starts_with = list(
+
+		)
+
+/obj/structure/closet/walllocker_double/north
```
Comment (KillianKirilenko) [inline code/game/objects/structures/crates_lockers/closets/walllocker.dm:244]: Forgot to subtype these so you're just overriding the ones further up.

Comment (GhostActual) [reply code/game/objects/structures/crates_lockers/closets/walllocker.dm:244]: This isn't supposed to be here. Will remove later. Tired.

Comment (GhostActual) [reply code/game/objects/structures/crates_lockers/closets/walllocker.dm]: Fixed

**Comment by BonniePandora** [comment]
Unless I'm missing something, did you mean to remove the gunbox_vr.dm?
That's still utilised in the current armory setups, it's not the detective's kind that spawns in there.
image

**Comment by GhostActual** [comment]
> Unless I'm missing something, did you mean to remove the gunbox_vr.dm? That's still utilised in the current armory setups, it's not the detective's kind that spawns in there. image

Will fix tomorrow, am tired

**Comment by Very-Soft** [comment]
For the things you removed or changed the paths of, please ensure that none of those things are actually present on any of the maps we use, including any offmaps or pois, and if they are, fix the paths. I had to do a lot of work to clean things up after you changed the ammo boxes, and I still don't think I got it all. UWU

**Comment by GhostActual** [comment]
> For the things you removed or changed the paths of, please ensure that none of those things are actually present on any of the maps we use, including any offmaps or pois, and if they are, fix the paths. I had to do a lot of work to clean things up after you changed the ammo boxes, and I still don't think I got it all. UWU

None of the paths got changed, and stuff was just moved around. Only the `Auto Webley` was removed. I found no spots in POIs that it spawned, only the random gun spawn code.

As for the ammo code, I thought I got them all in the POIs and it compiled with no error. Same here.

**Pull request merged**